compiler/modules/bir/bir.bal

```diff
@@ -210,11 +210,12 @@ public type Insn
     |BranchInsn|CondBranchInsn|CatchInsn|PanicInsn|ErrorConstructInsn;
 
 public type Operand ConstOperand|Register;
-public type SimpleConstOperand ()|boolean|int|float;
+public type SimpleConstOperand ()|boolean|int|float|decimal;
 public type ConstOperand SimpleConstOperand|string;
```

Comment:
Want to use new t:SingleValue define here.

Can we get rid of SingleConstOperand?

Reply:
Fixed in #769